### ISSUE TITLE
Far right column never gets filled

### ISSUE THREAD

**Issue by moltzaum** [ryanv3160/Team-9-Driver#18]
By the time the game is over, it looks like the below grid; the far right column never gets filled. This seems to just be an off-by-one error.
```
{
    "grid":[
        [1,1,2,0,2,2,1,0],
        [0,2,1,0,2,1,0,0],
        [0,1,2,0,1,2,0,0],
        [0,2,2,0,1,1,0,0],
        [0,0,2,0,1,0,0,0],
        [0,0,1,0,0,0,0,0],
        [0,0,0,0,0,0,0,0]
    ]
}
```

**Comment by moltzaum** [comment]
Update: we need to use column major. Also, the determine row function should iterate the opposite way.

**Comment by moltzaum** [comment]
This was fixed by #22
